tests/testthat/test-robustbayesianmetaanalysis.R

```diff
@@ -1213,106 +1213,106 @@ fitted_path <- file.path("robmaFit.RDS")
   })
   
   test_that("Information table results match", {
     table <- results[["results"]][["individual_models"]][["collection"]][["individual_models_model_12"]][["collection"]][["individual_models_model_12_temp_info"]][["data"]]
-    expect_equal_tables(table,
-                        list(0.176759472366701, -5.90832379591304, 0.0116467202823197, 0.0625
-                        ))
+    jaspTools::expect_equal_tables(table,
+                                   list(0.180259295733511, -2.59886113749133, 0.0118745860806326, 0.0625
+                                   ))
   })
   
   test_that("Priors table results match", {
     table <- results[["results"]][["individual_models"]][["collection"]][["individual_models_model_12"]][["collection"]][["individual_models_model_12_temp_priors"]][["data"]]
-    expect_equal_tables(table,
-                        list("Normal(0, 1)[-Inf, Inf]", "Two-sided((0.1, 0.05), (1, 1, 1))",
-                             "InvGamma(1, 0.15)[0, Inf]"))
+    jaspTools::expect_equal_tables(table,
+                                   list("Normal(0, 1)[-Inf, Inf]", "Two-sided((0.1, 0.05), (1, 1, 1))",
+                                        "InvGamma(1, 0.15)[0, Inf]"))
   })
   
   test_that("Estimated Studies' Effects (Î¸) table results match", {
     table <- results[["results"]][["individual_models"]][["collection"]][["individual_models_model_12"]][["collection"]][["individual_models_model_12_temp_studies"]][["data"]]
-    expect_equal_tables(table,
-                        list(0.00852024439881299, 735, -0.255923008947252, 0.176748964322513,
-                             0.167154704813216, 1.00091629944768, "Study 1", 0.663245585745701,
-                             0.00809948454702872, 749, -0.274041610154144, 0.160779535771706,
-                             0.157944785429571, 1.00068785496871, "Study 2", 0.612301087856998,
-                             0.00807508788266646, 709, -0.280653418186545, 0.136876373175994,
-                             0.134293585298117, 1.00066087953453, "Study 3", 0.561929530379696
-                        ))
+    jaspTools::expect_equal_tables(table,
+                                   list(0.00775853627019784, 855, -0.270720099051176, 0.170035632027558,
+                                        0.164009336133295, 1.00189805889533, "Study 1", 0.62568049934064,
+                                        0.00711458848006568, 905, -0.285009431407208, 0.147833903492011,
+                                        0.145923533777225, 1.00266486701686, "Study 2", 0.562315238784119,
+                                        0.00696160495950572, 888, -0.274416438770882, 0.128132867393647,
+                                        0.127580621830772, 1.00369316402907, "Study 3", 0.524550536497032
+                                   ))
   })
   
-  test_that("Estimated Weights (Ď‰) table results match", {
+  test_that("Estimated Weights (Ï‰) table results match", {
```

Comment:
symbols like `Ï‰` will probably make the R CMD CHECK [complain](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Encoding-issues) in the future.

Reply:
thanks, I missed this one.